### ISSUE TITLE
replacing the interface of irlbaSVD with CppIrlba `request`

### ISSUE THREAD

**Issue by xiangpin** [LTLA/BiocSingular#16]
Thanks for developing the infrastructure (`BiocSingular` and `CppIrlba`). They are significant in omics or other data analysis. However,
the `runIrlbaSVD` uses the `irlba`, which is slow for big data. Since `CppIrlba` has implemented the same work with `Eigen`, whether it will be applied to  `BiocSinglular`.

**Comment by LTLA** [comment]
Indeed, we did have plans for that, but the relevant proposal didn't get funded. :disappointed: 

So now there's no one to work on it. (Unfortunately, I don't have the time to improve this package; I just try to fix any bugs and keep things running as best as I can.) Back to the grant proposal drawing board, I guess.
